src/rws_subscription.cpp

```diff
@@ -70,9 +78,12 @@ namespace abb :: rws
       }
       catch (Poco::TimeoutException const&)
       {
+        now = std::chrono::steady_clock::now();
```

Comment:
Reusing variable for a different purpose (now it is a different "now"). I would use a scope-local variable with a different name.